tests/cross/src/test/scala/tests/hover/HoverTermSuite.scala

```diff
@@ -180,7 +184,9 @@ class HoverTermSuite extends BaseHoverSuite {
        |""".stripMargin.hover,
     compat = Map(
       "3" ->
-        "class Foo: Foo".hover
+        """|Foo[Int]
```

Comment:
You can remove this case as now it equals to scala2

Reply:
https://github.com/scalameta/metals/pull/3854/commits/e1956bf9efa8cdc0cf802c10ce92a14d313e7b47